.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - "2.7"
+  - "3.5"
+install:
+  - "pip install flake8"
+script:
+ - flake8 . --ignore E501,F841
```

Comment:
this will ignore long lines and unused variables see line 112 of record_audio.py

Reply:
Python 3.5?  Why not jump to 3.6?  (3.7 is in beta)

Reply:
good point. i copied an old travis file. will update to 3.6, shouldn't require any code changes